cli/dcoscli/data/help/node.txt

```diff
@@ -23,7 +23,8 @@ Options:
                             configuration, the private slaves are unreachable from the public
                             internet. You can access them using this option, which will first hop
                             from the publicly available master.
-    --slave=<slave-id>      Access the slave with the provided ID
+    --slave=<slave-id>      Deprecated. Please use --id.
+    --id=<node-id>          Access the node with the provided ID
```

Comment:
How about `Mesos ID`? As noted below, the ID is not quite a UUID, and I'm not sure how that information (the format of the ID) is relevant to the user anyway.